client/state/reader/tags/items/reducer.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * External dependencies
+ */
+import { keyBy, merge, mapValues } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	READER_TAGS_RECEIVE,
+	READER_UNFOLLOW_TAG_RECEIVE,
+} from 'state/action-types';
+import { createReducer, } from 'state/utils';
+
+/*
+ * since the api always returns the whole list of followed tags unpaginated, both read/tags*,
+ * we can do a full replace instead of merge
+ *
+ * the shape of a tag is { id, url, title, displayName, isFollowing }.
+ */
+export const items = createReducer( null, {
+	[ READER_TAGS_RECEIVE ]: ( state, action ) => {
+		const tags = action.payload;
+		const resetFollowingData = action.meta.resetFollowingData;
```

Comment:
yes. action creator ensures it: https://github.com/Automattic/wp-calypso/pull/11733/files#diff-79bab4e8d25693412ab1cee8bbc80ef2R38